jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListPresenterTest.java

```diff
@@ -140,4 +146,36 @@ public void testSetupDetailBreadcrumb() {
                                           eq(Commands.DO_NOTHING));
     }
 
+    @Test
+    public void bulkReleaseOnlyOnReservedOrInProgressTasksTest() {
+        List<TaskSummary> taskSummaries = new ArrayList<>();
+        taskSummaries.add(createTestTaskSummary(TASK_ID, TASK_STATUS_RESERVED, identity.getIdentifier()));
+        taskSummaries.add(createTestTaskSummary(TASK_ID + 1, TASK_STATUS_RESERVED, "otherUser"));
+        taskSummaries.add(createTestTaskSummary(TASK_ID + 2, TASK_STATUS_READY, ""));
+        taskSummaries.add(createTestTaskSummary(TASK_ID + 3, TASK_STATUS_IN_PROGRESS, ""));
+
+        getPresenter().bulkRelease(taskSummaries);
+
+        verify(taskService, times(3)).releaseTask(anyString(), eq(TASK_DEPLOYMENT_ID), anyLong());
+        verify(taskService).releaseTask(anyString(), eq(TASK_DEPLOYMENT_ID), eq(TASK_ID));
+        verify(taskService).releaseTask(anyString(), eq(TASK_DEPLOYMENT_ID), eq(TASK_ID + 1));
+        verify(taskService).releaseTask(anyString(), eq(TASK_DEPLOYMENT_ID), eq(TASK_ID + 3));
+    }
+
+    @Test
+    public void bulkSuspendOnlyOnReserveInProgressReadyTest() {
```

Comment:
It seems Suspend operation shouldn't be applicable to 'Ready' tasks. That's at least what I see in workbench.

Reply:
@jstastny-cz That condition is a bit different in "Manage/Tasks" https://github.com/kiegroup/jbpm-wb/blob/9bdc0a0d3cb80f8f9dc65e8e2afaa593bf355d05/jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListPresenter.java#L62 and "Task Inbox"..https://github.com/kiegroup/jbpm-wb/blob/9bdc0a0d3cb80f8f9dc65e8e2afaa593bf355d05/jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskListPresenter.java#L75

It's true for 'Task Inbox', only tasks with the status Reserved and InProgress can be suspended, but here, the status 'Ready' is allowed too. I have used the same conditions that were defined for the individual task actions